src/dd/Operations.cpp

```diff
@@ -88,7 +88,7 @@ void dumpTensor(qc::Operation* op, std::ostream& of,
     const auto localDD = getDD(localOp.get(), dd);
 
     // translate local DD to matrix
-    const auto localMatrix = localDD.getMatrix();
+    const auto localMatrix = dd.getMatrix(localDD, localQubits);
```

Comment:
This is using a method that has been deleted on the main branch. You'll find more details in other comments.